VideoAPI/VideoApi.Domain/Enums/VirtualCourtRoomType.cs

```diff
@@ -3,6 +3,7 @@ namespace VideoApi.Domain.Enums
     public enum VirtualCourtRoomType
     {
         JudgeJOH,
-        Participant
+        Participant,
+        WaitingRoom
```

Comment:
Do we need WaitingRoom here? We wouldn't be creating this room type

Reply:
Eventually the RoomType will be obsolete. Instead we will use the new Enum above. So yes it will need WR here.

Reply:
updated enum with number